context/brand-context/springernature/scss/10-settings/typography.scss

```diff
@@ -26,7 +26,7 @@ $context--font-size-h5: $context--font-size-md;
 $context--font-size-base: 1em;
 
 $context--line-height-h1: 2.625rem;
-$context--line-height-h2: 1.75rem;
+$context--line-height-h2: 2rem;
 $context--line-height-h3: 1.5rem;
 $context--line-height-h4: 1.5rem;
```

Comment:
@foxintherain Could we not make h3 line height `1.75em` or at least different to h4 line height?

Reply:
@aliuk2012 have made a new commit: https://github.com/springernature/frontend-toolkits/commit/f5bb67699dae57739f6e52a802128361dbd1d5db